tests/ts/su/environ

```diff
@@ -13,15 +13,19 @@ ts_skip_nonroot
 ts_check_test_command "$TS_CMD_SU"
 
 # all should be avalable
-ZUU=zuux FOO=abc BAR=baar $TS_CMD_SU --command 'echo foo:$FOO bar:$BAR zuu:$ZUU' >> $TS_OUTPUT
+ZUU=zuux FOO=abc BAR=baar $TS_CMD_SU --command 'echo foo:$FOO bar:$BAR zuu:$ZUU' >> $TS_OUTPUT 2>> $TS_ERRLOG
 
 # all should be removed
-ZUU=zuux FOO=abc BAR=baar $TS_CMD_SU --login --command 'echo foo:$FOO bar:$BAR zuu:$ZUU' >> $TS_OUTPUT
+ZUU=zuux FOO=abc BAR=baar $TS_CMD_SU --login --command 'echo foo:$FOO bar:$BAR zuu:$ZUU' >> $TS_OUTPUT 2>> $TS_ERRLOG
 
 # only specified should be avalable
-ZUU=zuux FOO=abc BAR=baar $TS_CMD_SU --whitelist-environment FOO --login --command 'echo foo:$FOO bar:$BAR zuu:$ZUU' >> $TS_OUTPUT
+ZUU=zuux FOO=abc BAR=baar $TS_CMD_SU --whitelist-environment FOO --login --command 'echo foo:$FOO bar:$BAR zuu:$ZUU' >> $TS_OUTPUT 2>> $TS_ERRLOG
 
 # only specified should be avalable
-ZUU=zuux FOO=abc BAR=baar $TS_CMD_SU --whitelist-environment FOO,BAR --login --command 'echo foo:$FOO bar:$BAR zuu:$ZUU' >> $TS_OUTPUT
+ZUU=zuux FOO=abc BAR=baar $TS_CMD_SU --whitelist-environment FOO,BAR --login --command 'echo foo:$FOO bar:$BAR zuu:$ZUU' >> $TS_OUTPUT 2>> $TS_ERRLOG
+
+
+grep -q 'Authentication failure' $TS_ERRLOG
```

Comment:
The error log needs to be commited, too? Or?

Reply:
I do not understand.  The goal is to skip the test if su(1) does not work as expected (usually due to PAM).

Reply:
I had the impression the the reference logfile would also need to be commited.
But apparently not, so disregard.